packages/gasket-plugin-morgan/README.md

```diff
@@ -53,7 +51,7 @@ Defaults:
 module.exports = {
   plugins: {
     add: [
-      '@gasket/plugin-log',
+      '@gasket/plugin-logger',
```

Comment:
With logger now built-in, I don't think we need to show it being added in the examples.

Reply:
DOH, missed this one.